### DISCUSSION TITLE
The most played music plugin should show how many times it was played.

### DISCUSSION THREAD

**Discussion by Milo123459** [lowlighter/metrics#663] [Ideas]
Pretty simple, I think it would be a nice addition.

**Comment by lowlighter** [comment]
It should be possible with the editions of @lhvy made in the `top played`, provided that the APIs expose the number of time a music has been played

**Reply by lhvy** [reply]
It depends on the API, last fm already does this, but Spotify does not as the API doesn't provide this information.

**Reply by Milo123459** [reply]
Oh, alright then.
